optional_plugins/html/avocado_result_html/__init__.py

```diff
@@ -248,8 +248,7 @@ def _render(self, result, output_path):
             else:
                 from pystache import view
                 v = view.View(template, context)
-                report_contents = v.render('utf8')  # encodes into ascii
-                report_contents = codecs.decode("utf8")  # decode to unicode
+                report_contents = v.render('utf8')
```

Comment:
Will this bring the 3154794 trello card back to life? I can't see the reference card in 3154794, so @ldoktor should take a look here.